src/EditorFeatures/Test/CodeFixes/ErrorCases/CodeFixExceptionInFixableDiagnosticIds2.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.CodeFixes.ErrorCases
+{
+    public class ExceptionInFixableDiagnosticIds2 : CodeFixProvider
+    {
+        public sealed override ImmutableArray<string> FixableDiagnosticIds => new ImmutableArray<string>();
```

Comment:
The test would still be doing that. However, as currently written it requires the reader to decipher the point of the test, which is to ensure that the handler throws an exception for `default(ImmutableArray<string>)` regardless of the manner/syntax in which that instance was obtained.